lib/mayaUsd/ufe/UsdAttributes.cpp

```diff
@@ -319,13 +319,14 @@ Ufe::UndoableCommand::Ptr UsdAttributes::removeAttributeCmd(const std::string& n
 {
     return UsdRemoveAttributeCommand::create(fItem, name);
 }
-
-Ufe::UndoableCommand::Ptr
-UsdAttributes::renameAttributeCmd(const std::string& targetName, const std::string& newName)
+#endif
+//#if (UFE_PREVIEW_VERSION_NUM >= 4033)
```

Comment:
Commented-out code.

Reply:
yes I forgot to remove the comment.

Reply:
The `#if` condition is needed.

Reply:
@JGamache-autodesk The code is not consistent in this file. Does it need the two conditions `UFE_V4_FEATURES_AVAILABLE` and `UFE_PREVIEW_VERSION_NUM >= 4033` -- or -- only the latter?